site/quickstarts/index.html

```diff
@@ -0,0 +1,178 @@
+<!--                                    <div class="quickstart-tech-title">Hanami</div>-->
+<!--                                </a>-->
+                                <a id="w-node-a56ccc9d-4da8-1bfc-63a9-a9183cd361d2-3d0acb82" href="/blog/2021/09/09/how-to-set-up-single-sign-on-between-fusionauth-joomla" class="docs-lp-link-block shadow-removed quickstart w-inline-block"><img src="/assets/img/docs/landing/php-joomla.svg" loading="lazy" alt="" class="quickstart-tech-icon">
+                                    <div class="quickstart-tech-title">Joomla</div>
+                                </a>
+                                <a id="w-node-_5a735e10-450a-94c0-bbd7-80a2c187b070-3d0acb82" href="/blog/2023/03/13/single-sign-on-laravel-fusionauth" class="docs-lp-link-block shadow-removed quickstart w-inline-block"><img src="/assets/img/docs/landing/laravel.svg" loading="lazy" alt="" class="quickstart-tech-icon">
+                                    <div class="quickstart-tech-title">Laravel</div>
+                                </a>
+<!--                                TODO next.js -->
+<!--                                <a id="w-node-_6a46af1b-e1aa-9946-a669-02de06f1d9af-3d0acb82" href="#" class="docs-lp-link-block shadow-removed quickstart w-inline-block"><img src="/assets/img/docs/landing/nextjs.svg" loading="lazy" alt="" class="quickstart-tech-icon">-->
+<!--                                    <div class="quickstart-tech-title">Next.js</div>-->
+<!--                                </a>-->
+                                <a id="w-node-_83379826-326b-a2b6-ef0f-5da8cd9b979f-3d0acb82" href="/blog/2020/05/13/setting-up-single-sign-on-for-nodebb" class="docs-lp-link-block shadow-removed quickstart w-inline-block"><img src="/assets/img/docs/landing/nodebb.svg" loading="lazy" alt="" class="quickstart-tech-icon">
+                                    <div class="quickstart-tech-title">NodeBB</div>
+                                </a>
+                                <a id="w-node-b4b38f89-4563-7817-46fe-caebafaca7a8-3d0acb82" href="/blog/2020/06/11/building-protected-api-with-rails-and-jwt" class="docs-lp-link-block shadow-removed quickstart w-inline-block"><img src="/assets/img/docs/landing/ruby-on-rails.svg" loading="lazy" alt="" class="quickstart-tech-icon">
+                                    <div class="quickstart-tech-title">Ruby On Rails</div>
+                                </a>
+                                <a id="w-node-a55ab806-0b3c-155c-76ee-2a86b19e296d-3d0acb82" href="/blog/2023/01/03/spring-and-fusionauth" class="docs-lp-link-block shadow-removed quickstart w-inline-block"><img src="/assets/img/docs/landing/spring.svg" loading="lazy" alt="" class="quickstart-tech-icon">
```

Comment:
```suggestion
                                <a id="w-node-a55ab806-0b3c-155c-76ee-2a86b19e296d-3d0acb82" href="/docs/v1/tech/tutorials/integrate-java-spring" class="docs-lp-link-block shadow-removed quickstart w-inline-block"><img src="/assets/img/docs/landing/spring.svg" loading="lazy" alt="" class="quickstart-tech-icon">
```

Should prefer the docs where they exist. This is my fault for not updating the spreadsheet.